src-tauri/src/main.rs

```diff
@@ -36,37 +46,60 @@ fn main() {
             app.emit_all("single-instance", Payload { args: argv, cwd })
                 .unwrap();
         }))
-        .plugin(
-            tauri_plugin_log::Builder::default()
-                .targets([LogTarget::LogDir, LogTarget::Stdout])
-                .build(),
-        )
         .plugin(tauri_plugin_fs_watch::init())
-        .plugin(tauri_plugin_window_state::Builder::default().build())
+        // .plugin(tauri_plugin_window_state::Builder::default().build())
         .plugin(tauri_plugin_store::Builder::default().build())
-        .setup(|app| {
-            // Add window shadows
-            let window = app.get_window("main").unwrap();
-            set_shadow(&window, true).expect("Unsupported platform!");
-            Ok(())
-        })
+        .plugin(cohdb::auth::init(
+            "kHERjpU_rXcvgvLgwPir0w3bqcgETLOH-p95-PVxN-M".to_string(),
```

Comment:
Need to investigate a better way to inject this than having it in the code. It's technically not a secret so it's not the end of the world, but still, not best practice.

Reply:
I also have in coh2stats and coh3stats some "secrets" publicly , I think it's the cost we have to pay in case we want to have this open source, I think it's better to have it this way - so anyone can clone, run and contribute. 

But also if this is only for COHDB integration to work - and the app can run without it I think we could hide it into secrets on GitHub.

Reply:
Yeah that's fair. This is just the OAuth Client ID so it's technically a public value, definitely fine to keep here.